bookbotproject/conversationtree/admin.py

```diff
@@ -18,7 +18,14 @@
                     list_display=('template', 'node',),
                     list_display_links=('template',)
                     ,)
+admin.site.register(Book,
+                    admin.ModelAdmin,
+                    list_filter=('author', ),
+                    list_display=('__unicode__', 'author'),
+                    list_display_links=('__unicode__',)
+                    ,)
 admin.site.register(Comment,
                     admin.ModelAdmin,
-                    list_display=('__unicode__', 'user', 'comment'),
+                    list_filter=('user', 'book'),
+                    list_display=('__unicode__', 'user', 'comment', 'book'),
                                        list_display_links=('__unicode__',),)
```

Comment:
Indentations are important :) It is not a big deal, you can make the change next time you commit.

Reply:
You are right! Fixed it now. ^^